Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.py

```diff
@@ -147,27 +147,8 @@ def main():  # pragma: no cover
     args = demisto.args()
     args["integration_context_brand"] = INTEGRATION_CONTEXT_BRAND
     args["integration_name"] = INTEGRATION_NAME
-    api_key = demisto.params().get('apikey')
-    api_key_id = demisto.params().get('apikey_id')
-    url = demisto.params().get('url')
+    url = "/api/webapp/"
```

Comment:
what if the build number is lower than 924770?

Reply:
i'll add the check here as well